test/runnable/b18034.d

```diff
@@ -0,0 +1,23 @@
+version (D_SIMD)
```

Comment:
Should this have `// REQUIRED_ARGS: -O`  or maybe both variants with `ARG_SETS`?

Reply:
Good catch, I forgot about the `-O`.